portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Joey's Image Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Image Gallery</h1>
+      <p>Click <a href="index.html">here</a> to see return home.</p>
```

Comment:
I assume this is supposed to be "to return home" or "to see the home page" ?

Reply:
Yes, it's supposed to be "to return home"